middleware/grpc_logging.go

```diff
@@ -24,3 +24,17 @@ var ServerLoggingInterceptor = func(ctx context.Context, req interface{}, info *
 	}
 	return resp, err
 }
+
+// TruncatedServerLoggingInterceptor logs gRPC requests, errors and latency,
+// but omits request bodies.
+var TruncatedServerLoggingInterceptor = func(ctx context.Context, req interface{}, info *grpc.UnaryServerInfo, handler grpc.UnaryHandler) (interface{}, error) {
```

Comment:
"Truncated" suggests to me that it will keep the first N bytes and omit the rest.  

How about making this a member function on an object with `withRequest` bool field?

Reply:
Sorry to be slow, but I'm not sure where best to put said member function?

Reply:
In this file.  So what are currently `ServerLoggingInterceptor` and `TruncatedServerLoggingInterceptor` would be just one member on an object with the config.

Something like `Log` at https://github.com/weaveworks/common/blob/master/middleware/logging.go#L15-L17